alert-database/src/main/java/com/synopsys/integration/alert/database/api/StaticJobAccessor.java

```diff
@@ -0,0 +1,276 @@
+import com.synopsys.integration.alert.database.job.JobConfigurationModelFieldExtractorUtils;
+import com.synopsys.integration.alert.database.job.JobConfigurationModelFieldPopulationUtils;
+import com.synopsys.integration.alert.database.job.azure.boards.AzureBoardsJobDetailsAccessor;
+import com.synopsys.integration.alert.database.job.azure.boards.AzureBoardsJobDetailsEntity;
+import com.synopsys.integration.alert.database.job.blackduck.BlackDuckJobDetailsAccessor;
+import com.synopsys.integration.alert.database.job.blackduck.BlackDuckJobDetailsEntity;
+import com.synopsys.integration.alert.database.job.email.EmailJobDetailsAccessor;
+import com.synopsys.integration.alert.database.job.email.EmailJobDetailsEntity;
+import com.synopsys.integration.alert.database.job.jira.cloud.JiraCloudJobDetailsAccessor;
+import com.synopsys.integration.alert.database.job.jira.cloud.JiraCloudJobDetailsEntity;
+import com.synopsys.integration.alert.database.job.jira.server.JiraServerJobDetailsAccessor;
+import com.synopsys.integration.alert.database.job.jira.server.JiraServerJobDetailsEntity;
+import com.synopsys.integration.alert.database.job.msteams.MSTeamsJobDetailsAccessor;
+import com.synopsys.integration.alert.database.job.msteams.MSTeamsJobDetailsEntity;
+import com.synopsys.integration.alert.database.job.slack.SlackJobDetailsAccessor;
+import com.synopsys.integration.alert.database.job.slack.SlackJobDetailsEntity;
+
+@Component
+public class StaticJobAccessor implements JobAccessor {
+    private final DistributionJobRepository distributionJobRepository;
+    private final BlackDuckJobDetailsAccessor blackDuckJobDetailsAccessor;
+    private final AzureBoardsJobDetailsAccessor azureBoardsJobDetailsAccessor;
+    private final EmailJobDetailsAccessor emailJobDetailsAccessor;
+    private final JiraCloudJobDetailsAccessor jiraCloudJobDetailsAccessor;
+    private final JiraServerJobDetailsAccessor jiraServerJobDetailsAccessor;
+    private final MSTeamsJobDetailsAccessor msTeamsJobDetailsAccessor;
+    private final SlackJobDetailsAccessor slackJobDetailsAccessor;
+
+    // Temporary until all three tiers of the application have been updated to new Job models
+    private final RegisteredDescriptorRepository registeredDescriptorRepository;
+    // BlackDuck is currently the only provider, so this is safe in the short-term while we transition to new models
+    private final ProviderKey blackDuckProviderKey;
+
+    @Autowired
+    public StaticJobAccessor(
+        DistributionJobRepository distributionJobRepository,
+        BlackDuckJobDetailsAccessor blackDuckJobDetailsAccessor,
+        AzureBoardsJobDetailsAccessor azureBoardsJobDetailsAccessor,
+        EmailJobDetailsAccessor emailJobDetailsAccessor,
+        JiraCloudJobDetailsAccessor jiraCloudJobDetailsAccessor,
+        JiraServerJobDetailsAccessor jiraServerJobDetailsAccessor,
+        MSTeamsJobDetailsAccessor msTeamsJobDetailsAccessor,
+        SlackJobDetailsAccessor slackJobDetailsAccessor,
+
```

Comment:
Do we want this space here? We already have the space in the class members, so I think we should remove this blank line.